src/canvas.class.js

```diff
@@ -1200,6 +1200,7 @@
      */
     dispose: function () {
       var wrapper = this.wrapperEl;
+      fabric.runningAnimations.cancelAll();
```

Comment:
this needs to be executed per canvas.
EIther you pass in the canvas as argument and cancelAll becomes a cancelAll by canvas, or in an application with 2 canvases this would stop everything.

Reply:
Good point.
When an objects animates does it have to be attached to a canvas? I guess not. So I'm not sure how to do what you suggest.
I can add a canvas property to the animation context object but what if there's no canvas set to the object??
Or maybe we should iterate over all running animations and check at that moment who is attached to the canvas arg and cancel that?